libs/wire-subsystems/src/Wire/TeamCollaboratorsSubsystem/Interpreter.hs

```diff
@@ -93,6 +94,43 @@ getAllTeamCollaboratorsImpl zUser team = do
   guardPermission (tUnqualified zUser) team TeamMember.NewTeamCollaborator InsufficientRights
   Store.getAllTeamCollaborators team
 
+removeTeamCollaboratorImpl ::
+  ( Member TeamSubsystem r,
+    Member (Error TeamCollaboratorsError) r,
+    Member Store.TeamCollaboratorsStore r,
+    Member Now r,
+    Member NotificationSubsystem r
+  ) =>
+  Local UserId ->
+  UserId ->
+  TeamId ->
+  Sem r ()
+removeTeamCollaboratorImpl zUser user team = do
+  guardPermission (tUnqualified zUser) team TeamMember.RemoveTeamCollaborator InsufficientRights
+  Store.removeTeamCollaborator user team
+  -- TODO gdf remove O2O conversations
```

Comment:
I'm not sure how to handle this one.

I have tried to move `services/galley/src/Galley/Effects/ConversationStore.hs` in `lib/wire-subsystems`, but Cassandra queries and many data-types are in Galley.

I'm also not sure how to list the O2O conversations of a `(TeamId, UserId)`.

I would be inclined to create a dedicate effects, with one operation and a Cassandra interpreter, having queries in the same file.

WDYT?